### PR TITLE
Mention Translate Behavior in Internationalization & Localization

### PR DESCRIPTION
**Pull request by LordSimal**

### CODE REVIEW

File: en/core-libraries/internationalization-and-localization.rst [hunk 1]
```diff
@@ -632,3 +632,8 @@ option to restrict which locales will automatically be used.
 .. meta::
     :title lang=en: Internationalization & Localization
     :keywords lang=en: internationalization localization,internationalization and localization,language application,gettext,l10n,pot,i18n,translation,languages
+    
+Translate Content/Entities
+=======================================================
```
Comment (dereuromark) [inline en/core-libraries/internationalization-and-localization.rst]: This should be the same length as your heading

Comment (LordSimal) [reply en/core-libraries/internationalization-and-localization.rst]: ohhhhhh, so its not a fixed amount of = but it is related to the length of the heading... got it

File: en/core-libraries/internationalization-and-localization.rst [hunk 2]
```diff
@@ -632,3 +632,8 @@ option to restrict which locales will automatically be used.
 .. meta::
     :title lang=en: Internationalization & Localization
     :keywords lang=en: internationalization localization,internationalization and localization,language application,gettext,l10n,pot,i18n,translation,languages
+    
+Translate Content/Entities
+==========================
+
+If you want to translate content/entities then you should look at the :doc:`Translate Behavior </orm/behaviors/translate>`
```
Comment (ndm2) [inline en/core-libraries/internationalization-and-localization.rst:635]: This should probably go _above_ the page's metadata information.

Comment (LordSimal) [reply en/core-libraries/internationalization-and-localization.rst:635]: I'm sorry but I dont know what you mean 😅

Comment (ndm2) [reply en/core-libraries/internationalization-and-localization.rst:635]: The `.. meta::` directive belongs to the page, not the "*Automatically Choosing the Locale Based on Request Data*" section, so your new section should be placed above the directive.

Comment (LordSimal) [reply en/core-libraries/internationalization-and-localization.rst:635]: oh sh*t, yea that seems logical 😓

**Pull request merged**